lib/srv/db/common/iam/aws.go

```diff
@@ -113,6 +137,33 @@ func getRedshiftPolicyDocument(db types.Database) (*awslib.PolicyDocument, Place
 	return policyDoc, placeholders, nil
 }
 
+// getRedshiftServerlessPolicyDocument returns the policy document used for
+// Redshift Serverless databases.
+//
+// https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonredshiftserverless.html
+func getRedshiftServerlessPolicyDocument(db types.Database) (*awslib.PolicyDocument, Placeholders, error) {
```

Comment:
seems that err != nil is never returned: 
```suggestion
func getRedshiftServerlessPolicyDocument(db types.Database) (*awslib.PolicyDocument, Placeholders) {
```

Reply:
I think I will keep this one as it is. The parent function returns all 3 parameters as an one-liner `return getRedshiftServerlessPolicyDocument(db)`. I think it's a little easier to read this way.